apis/monitor/v1alpha1/zz_metricalert_terraformed.go

```diff
@@ -54,6 +54,14 @@ func (tr *MetricAlert) SetObservation(obs map[string]interface{}) error {
 	return json.TFParser.Unmarshal(p, &tr.Status.AtProvider)
 }
 
+// GetID returns ID of underlying Terraform resource of this MetricAlert
```

Comment:
The `MetricAlert` resource was implemented with the following PR: https://github.com/crossplane-contrib/provider-jet-azure/pull/94

While implementing the `MetricAlert` resource, we observed that, it has `ActionGroup` resource id reference. However, as there was no resource id reference support in `Terrajet`, this reference was not configured for this field.

It seems that with this PR, we will now be able to use `ActionGroup` id reference in the `MetricAlert` resource, right?

Reply:
Yes, that's right. Let's get this merged and we can add/configure/exemplify with a follow-up PR.